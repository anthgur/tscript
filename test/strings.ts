print "Hello World!";
print 'Hello again!';
print "Hello 'again'";
print 'Hello "again"';
print "I am on two\nlines";