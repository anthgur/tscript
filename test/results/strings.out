Hello World!
Hello again!
Hello 'again'
Hello "again"
I am on
two lines


Strings plus strings
